Frameworks/MetalPetal/Kernels/MTIComputePipelineKernel.m

```diff
@@ -156,6 +156,9 @@ - (MTIImagePromiseRenderTarget *)resolveWithContext:(MTIImageRenderingContext *)
         }
     }
     
+    #if TARGET_OS_TV
```

Comment:
How about using #if TARGET_OS_TV to control the `supportsNonUniformThreadgroupSize` below?

Reply:
Oh this does not work because the api is not available on tvOS. So it has to be a complier directive here.

Reply:
Yeah, although there might be a way to make this block a bit more readable.